lib/vdsm/clientIF.py

```diff
@@ -451,6 +451,9 @@ def prepareVolumePath(self, drive, vmId=None, path=None):
                             # not speicfy the type, we must use 'file'.
                             drive['diskType'] = DISK_TYPE.FILE
                     volPath = res['path']
+            # Managed drives are prepared in ManagedVolume.attach_volume
+            elif drive.get("managed", False):
+                volPath = drive['path']
```

Comment:
We discussed moving this first - before handling images. Any reason to leave it here or you just forgot to move it?

Reply:
I forgot :)